src/permission/rule-cache.js

```diff
@@ -10,7 +10,7 @@
 const RuleCache = function (options) {
   this._options = options
   this._data = {}
-  setInterval(this._purge.bind(this), options.cacheEvacuationInterval)
+  this._interval = setInterval(this._purge.bind(this), options.cacheEvacuationInterval)
```

Comment:
This should not affect anything _but_ the server can at least be closed properly by the caller by doing `clearInterval()` from the outside.